gravitee-apim-console-webui/src/management/api/documentation-v4/api-documentation-v4.component.html

```diff
@@ -22,11 +22,12 @@
     [isReadOnly]="isReadOnly"
     [hasPages]="pages && pages.length > 0"
     (onAddFolder)="addFolder()"
```

Comment:
Non-blocking comment: Another action could be to change all the `onXX` events to just `XX`, as you did with `onAddPage` -> `addPage`. Best practice is to have the output as a simple verb and the receiving function as the same simple verb or `on` + simple verb.

More info [here](https://angular.dev/style-guide#avoid-aliasing-inputs-and-outputs).

Reply:
Done